ResearchKit/Common/ORKScaleSliderView.m

```diff
@@ -251,7 +281,6 @@ - (instancetype)initWithFrame:(CGRect)frame {
         
         self.leftRangeLabel = [[ORKScaleRangeLabel alloc] initWithFrame:CGRectZero];
         self.leftRangeLabel.textAlignment = NSTextAlignmentCenter;
```

Comment:
For consistency, can you move this label creation code into `initWithFormatProvider:`?